### PR TITLE
Fix failing SetCookies_Success test

### PR DESCRIPTION
**Pull request by ViktorHofer**
See https://github.com/dotnet/runtime/issues/53931

### CODE REVIEW

**Comment by akoeplinger** [comment]
The reason why it fails is because of this:

https://github.com/dotnet/runtime/blob/78579ef0e7283fd70c410a2d2b0fb9dae6c9d191/src/libraries/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs#L132

Can you bump the Expires to some later date? 😄

**Comment by ViktorHofer** [comment]
WHAT THA HELL. Best test failure EVER 😂😂😂

**Review by akoeplinger — APPROVED** [review]

File: src/libraries/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs [hunk 1]
```diff
@@ -125,11 +125,11 @@ public static IEnumerable<object[]> SetCookiesData()
             yield return new object[]
             {
                 u,
-                "name98=value98; path=/; domain=.uri.com; expires=Wed, 09 Jun 2021 10:18:14 GMT, name99=value99",
+                "name98=value98; path=/; domain=.uri.com; expires=Wed, 09 Jun 2050 10:18:14 GMT, name99=value99",
```
Comment (stephentoub) [inline src/libraries/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs:128]: I'll be really excited if this test is still running in 29 years and fails then.

**Review by stephentoub — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Merging to unblock CI.

**Pull request merged**